### DISCUSSION TITLE
Add Overlay-Button to Map

### DISCUSSION THREAD

**Discussion by G4rdener** [streetcomplete/StreetComplete#4525] [Ideas]
As the title says, I would suggest quick access to the overlay menu. With the upcoming version 48 there will be two more overlays and switching between them should be much faster.
It would be best to display the overlay button directly on the map (like menu, compass, +, - and location) so you can switch between them much faster. Also clicking the ok button after selecting an overlay would then no longer be necessary in my opinion.

I can understand that the overlay function is not for everyone, especially beginners. Maybe it would be a possibility that everyone can decide for themselves if they want to have the button (maybe disabled by default).

I have created a small mockup (not very good). I also added the overlays to show how a quick switch between the overlays could look like, should they be visible on the map. 

![Screenshot_20221018-115704](https://user-images.githubusercontent.com/49069311/196403932-320cee6c-c798-4ada-a3d6-85f2c4ff4f5e.png)

**Comment by riQQ** [comment]
See also #4512
